tests/testthat/test-translate-sql-quantile.R

```diff
@@ -1,14 +1,14 @@
 test_that("quantile and median don't change without warning", {
-  expect_snapshot(translate_sql(quantile(x, 0.75), window = FALSE))
-  expect_snapshot(translate_sql(quantile(x, 0.75), vars_group = "g"))
+  expect_snapshot(translate_sql(quantile(x, 0.75, na.rm = TRUE), window = FALSE))
+  expect_snapshot(translate_sql(quantile(x, 0.75, na.rm = TRUE), vars_group = "g"))
   expect_snapshot(translate_sql(median(x, na.rm = TRUE), window = FALSE))
   expect_snapshot(translate_sql(median(x, na.rm = TRUE), vars_group = "g"))
 })
 
-test_that("median functions warn once if na.rm = FALSE", {
-  expect_warning(translate_sql(median("x")), "Missing values")
-  expect_warning(translate_sql(median("x")), NA)
-  expect_warning(translate_sql(median("x", na.rm = TRUE)), NA)
+test_that("quantile functions warn once if na.rm = FALSE", {
```

Comment:
Did you mean to remove the test for `median()` ?

Reply:
`median` is translated with `sql_median()` which is basically just a wrapper around `sql_quantile()`. So, replacing the tests for `median()` with tests for `quantile()` should be okay.